testsuite/HOWTO.md

```diff
@@ -32,8 +32,9 @@ It is recommended that you run the tests from the [git source repository](Source
 cd testsuite/MDAnalysisTests
 pytest  --disable-pytest-warnings
 ```
+Note: We use the `--disable-pytest-warnings` flag to currently suppress warnings about `yeild` based tests. Work in being done to fix this and you shouldn't need the flag if you are running a singe file/test.
```

Comment:
Actually we have a lot more warnings then that. They are expected so we do make them less noisy. But we should only recommend them when running large portions of the tests.